package.json

```diff
@@ -42,10 +42,10 @@
     "@remix-run/testing": "^2.13.1",
     "@types/js-cookie": "^3.0.6",
     "@types/lodash.debounce": "^4.0.9",
-    "@types/react": "^18.3.12",
-    "@types/react-dom": "^18.3.1",
-    "@typescript-eslint/eslint-plugin": "^8.12.1",
-    "@wixc3/define-remix-app": "^4.5.0",
+    "@types/react": "^18.3.11",
+    "@types/react-dom": "^18.3.0",
+    "@typescript-eslint/eslint-plugin": "^8.8.0",
+    "@wixc3/define-remix-app": "^4.5.1",
```

Comment:
Only `"@wixc3/define-remix-app"` should be bumped, the rest should be left alone.

Reply:
oops, bad merge.